_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,154 @@
+
+```bash
+opensearch-benchmark -h
+```
+{% include copy.html %}
+
+OpenSearch Benchmark will exit with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
+{: .warning}
+
+Now that OpenSearch Benchmark is installed on your host you can learn about [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/).
+
+## Installing with Docker
+
+You can find the official Docker images for OpenSearch Benchmark on [Docker Hub](https://hub.docker.com/r/opensearchproject/opensearch-benchmark) or on the [Amazon ECR Public Gallery](https://gallery.ecr.aws/opensearchproject/opensearch-benchmark).
+
+To pull the image from Docker Hub:
+
+```bash
+docker pull opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+To pull the image from Amazon ECR:
+
+```bash
+docker pull public.ecr.aws/opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+### Running Benchmark with Docker
+
+To run OpenSearch Benchmark, you use `docker run` to launch a container. OpenSearch Benchmark subcommands are passed as arguments when you start the container. OpenSearch Benchmark then processes the command, then stops the container after the requested operation completes.
+
+For example, the following command will print the help text for OpenSearch Benchmark to the command line, and then stop the container:
+
+```bash
+docker run opensearchproject/opensearch-benchmark -h
+```
+{% include copy.html %}
+
+OpenSearch Benchmark will exit with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
+{: .warning}
+
+### Establishing volume persistence in a Docker container
+
+To make sure your benchmark data and logs persist after your Docker container stops, specify a Docker volume to mount to the image when you work with OpenSearch Benchmark.
+
+Use the `-v` option and specify a local directory to mount, and a directory in the container where the volume is attached.
+
+The following example command creates a volume in a user's home directory, mounts the volume to the OpenSearch Benchmark container at `/opensearch-benchmark/.benchmark`, and then executes a test benchmark using the geonames workload. Some client options are also specified:
+
+```bash
+run -v $HOME/benchmarks:/opensearch-benchmark/.benchmark opensearchproject/opensearch-benchmark execute_test --target-hosts https://198.51.100.25:9200 --pipeline benchmark-only --workload geonames --client-options basic_auth_user:admin,basic_auth_password:admin,verify_certs:false --test-mode
+```
+{% include copy.html %}
+
+See [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/) to learn more about the files and subdirectories located in `/opensearch-benchmark/.benchmark`.
```

Comment:
`/opensearch-benchmark/.benchmark` can be switched to `~/.benchmark` as we have updated the Docker image to use PyPi and removed the need for `/opensearch-benchmark` venv.